src/RulesTest/MonsterTest.cs

```diff
@@ -236,6 +239,17 @@ public void MonsterUserControl()
             CheckForExtraRules(expectedResults);
         }
 
+        private static void RemoveEnglishSpecificRulesIfNeeded(Dictionary<RuleId, EvaluationCode> expectedResults)
+        {
+            if (!string.Equals(CultureInfo.CurrentCulture.ThreeLetterISOLanguageName, "eng", System.StringComparison.OrdinalIgnoreCase))
+            {
+                if (expectedResults.ContainsKey(RuleId.LocalizedControlTypeReasonable))
```

Comment:
Is there a common place where we can keep a list of English-specific rules/a function to remove them, usable in all of our system tests? Does this matter?

Reply:
So far this is the only place that we use them. I've refactored the single value into an `IEnumerable` to streamline the code a little bit. If a future change requires sharing of the `IEnumerable`, it'll be a trivial change to point to the new location.